data_steward/analytics/table_metrics/other_scripts/table_metrics_part_1.py

```diff
@@ -82,7 +87,7 @@ def cstr(s, color='black'):
         "ipmc_uchicago", "aouw_mcri", "syhc", "cpmc_ceders", "seec_ufl",
         "saou_uab", "trans_am_baylor", "cpmc_ucsd", "ecchc", "chci", "aouw_uwh",
         "cpmc_usc", "hrhc", "ipmc_northshore", "chs", "cpmc_ucsf", "jhchc",
-        "aouw_mcw", "cpmc_ucd", "ipmc_rush"
+        "aouw_mcw", "cpmc_ucd", "ipmc_rush", "va", "saou_umc"
```

Comment:
Do you want to consider using the `bq_utils.get_hpo_info()` for this? I think it helps to avoid hardcoding these, in case a new site is added.

Reply:
I think that would be better in future iterations. This is what Bernes did past iterations of the script so I stuck with the convention because I did not know that such a function existed. Perhaps in a future PR I can improve this.